src/Commands/WaPanel.php

```diff
@@ -0,0 +1,221 @@
+                    ];
+                }
+                if ($widget->type === 'teaser_list' && $widget->category_id) {
+                    $teaserListWidget = [
+                        'acf_fc_layout'  => $widget->type,
+                        'display_format' => $widget->display_format,
+                        'category' =>  WpTerm::id_from_whitealbum_id($widget->category_id),
+                        'sort_by' => 'custom',
+                        AcfName::FIELD_TEASER_AMOUNT => $widget->number_of_contents,
+                        AcfName::FIELD_SKIP_TEASERS_AMOUNT => $sortBySkipAmount[$widget->category_id] ?? 0,
+                    ];
+                    if (!isset($sortBySkipAmount[$widget->category_id])) {
+                        $sortBySkipAmount[$widget->category_id] = $widget->number_of_contents;
+                    } else {
+                        $sortBySkipAmount[$widget->category_id] += $widget->number_of_contents;
+                    }
+                    return $teaserListWidget;
+                }
+                if ($widget->type === 'newsletter') {
+                    return [
+                        'acf_fc_layout'  => $widget->type
+                    ];
+                }
+                return null;
+            })
+            ->rejectNullValues()
+            ->push([
+                'acf_fc_layout'  => 'teaser_list',
+                'display_format' => 'default',
+                'sort_by' => 'custom',
+                AcfName::FIELD_TEASER_AMOUNT => 12,
+                AcfName::FIELD_SKIP_TEASERS_AMOUNT => $sortBySkipAmount[WpTerm::whiteablum_id($category->term_id)],
+                'pagination' => true,
+                'category' => $category->term_id
+            ]);
+
+        update_field('field_5bb318f2ffcef', $content->toArray(), $category);
+    }
+
+    private function disableHooks()
+    {
+        // Disable generation of image sizes on import to speed up the precess
+        add_filter('intermediate_image_sizes_advanced', function ($sizes) {
+            return [];
+        });
+
+        // Disable on save hook to prevent call to content hub, Cxense and Bonnier Cache Manager
+        remove_action('save_post', [WpComposite::class, 'on_save'], 10);
+        remove_action('publish_to_publish', [BonnierCachePost::class, 'update_post'], 10);
+        remove_action('draft_to_publish', [BonnierCachePost::class, 'publishPost'], 10);
+        remove_action('transition_post_status', [CxensePost::class, 'post_status_changed'], 10);
+    }
+
+    private function fixFaultyImageFormats($content)
+    {
+        if ($content->type === 'image'
+            && ($extension = pathinfo($content->url, PATHINFO_EXTENSION))
+            && in_array($extension, ['psd'])
+        ) {
+            // If we find the image extension to be in the blacklist then we tell imgix to return as png format
+            $content->url .= '?fm=png';
```

Comment:
Hvorfor ikke jpeg? Jpeg fylder generelt mindre end png'er - især for billeder som ikke er grafik, men fotografier.

Reply:
Jeg kunne forstille mig at det var fordi at jeg ikke kan vide om der er noget som kræver alpha channel i pdf'en altså transparency hvilket jpeg jo ikke supporterer.

Reply:
Ahh, det giver mening :-)